coalib/coala_delete_orig.py

```diff
@@ -1,21 +1,20 @@
+import logging
 import os
 
```

Comment:
commit message body:
1. "uses " -> "use"
2. I would add the fact that you deprecate the `log_printer` param :)